.github/mistaken-pull-closer.yml

```diff
@@ -0,0 +1,16 @@
+# The JSONPath filter expression used to identify which PRs to close.
+# The data filtered is the pull request data along with other metadata passed in
+# by probot.
+# Default behavior: Close all PRs.
+filters:
+  - true
```

Comment:
If we merge this, we won't be able to modify this file again (the pull request would be closed :rofl:). Perhaps we want to add filters?

Reply:
In this repo we can always make a direct commit, they are not forbidden.

Reply:
After digging, it doesn't look like there's a way to filter by usergroup or a similar sensible attribute.